app/index.md

```diff
@@ -4,23 +4,11 @@ title: "A design history for your GOV.UK service"
 description: "Set up a place for you to document designs for your service"
 pagination:
   data: collections.all
-  reverse: true
```

Comment:
I wonder if we want to leave this in some-how, as it makes the most sense for people forking this repo. 🤔

Reply:
True, but then the introductory posts are in the wrong order. Maybe we leave it in the frontmatter but commented out?

Reply:
Fake the dates on the introductory posts so they are in the correct order?